### PR TITLE
Api

### PR DESCRIPTION
**Pull request by joshua-fogg**
Add routes
Currently using mock data sets for get messages / people routes.
Crated mock db.js for testing

### CODE REVIEW

File: server/apiRoutes.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+const express = require('express')
```
Comment (don-smith) [inline server/apiRoutes.js]: Please rename this file to `api-routes.js`. We use kabab-case for all filenames.

File: server/apiRoutes.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+const bodyParser = require('body-parser')
+
+const router = express.Router()
+
+const db = require('./db')
```
Comment (don-smith) [inline server/apiRoutes.js]: Let's move this line up so all the `require`s are together. Preferably with a blank line after the external `require`s.

File: server/apiRoutes.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+router.post('/messages', (req, res) => {
+  const message = {
+    senderId: req.body.senderId,
+    recipientId: req.body.recipientId,
+    imageUrl: req.body.imageUrl
+  }
+  db.createMessage(message)
+    .end((err, res) => {
+      if (err) {
+        res.status(500).err.message
```
Comment (don-smith) [inline server/apiRoutes.js]: I don't think this is right. I think you mean `res.status(500).send(err.message)`.

File: server/server.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
 var path = require('path')
 var express = require('express')
 
-var redditRoutes = require('./reddit')
+var apiRoutes = require('./apiRoutes')
```
Comment (don-smith) [inline server/server.js]: This will need to reflect the new file name.

**Comment by joshua-fogg** [comment]
integrated feedback

**Pull request merged**